lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -125,6 +127,20 @@ defmodule Membrane.MP4.Muxer.CMAF do
   def handle_process(Pad.ref(:input, _id) = pad, sample, ctx, state) do
     use Ratio, comparison: true
 
+    if state.last_pts > sample.pts do
+      Membrane.Logger.warn(
+        "PTS are out of order. Current PTS: #{sample.pts} Previous PTS: #{state.last_pts}"
```

Comment:
I'd point out that this probably indicates B-frames which are not supported

Reply:
done